packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -414,22 +413,22 @@ class AnimationController extends Animation<double>
   /// and the most recent tick of the animation.
   ///
   /// If the controller is not animating, the last elapsed duration is null.
-  Duration get lastElapsedDuration => _lastElapsedDuration;
-  Duration _lastElapsedDuration;
+  Duration get lastElapsedDuration => _lastElapsedDuration!;
```

Comment:
This is conflicting with the doc comment above it.

Reply:
Fixed